Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -28,6 +28,11 @@ final class SitterProfileViewModel: ObservableObject {
     /// Indicating the status of the saving operation.
     @Published var isSavingData = false
 
+    /// Indicates if the mandatory fields are empty.
+    var mandatoryFieldsAreEmpty: Bool {
+        name.isEmpty || phone.isEmpty
+    }
+
```

Comment:
Why does it appear in this PR, it has already been in the main, I think

Reply:
I'll try to fix it